Vic2ToHoI4/Source/HOI4World/HoI4Country.cpp

```diff
@@ -956,3 +987,15 @@ double HoI4::Country::calculateInfluenceFactor()
 		return std::clamp(influenceFactor, 1.0, 100.0);
 	}
 }
+
+bool HoI4::Country::canDeclareWar(const std::string& target)
```

Comment:
Is this used anywhere? It looks handy, but maybe wait to submit it at the same time as code that uses it.